packages/competence/bin/static/fragments/frame-admin-config.html

```
@@ -78,6 +78,70 @@ <h2 class="competence-admin-section-title" x-text-label="interface.admin.editors
+                                            <li class="competence-admin-drift-entry">
+                                                <span class="ti-tag" x-text="driftKindLabel(entry)"></span>
+                                                <span x-text="driftEntryText(entry)"></span>
+                                            </li>
+                                        </template>
+                                    </ul>
+                                </template>
+                            </div>
+                        </template>
+
+                        <div class="competence-admin-drift-apply">
+                            <input type="text" class="ti-input" x-model="driftNote"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)